scripts/with-java.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# 4, because that's the number of JVMs we test on
+mod="$((${GO_PIPELINE_COUNTER} % 4))"
+
+function use_jdk() {
+  source "${HOME}/.jabba/jabba.sh"
```

Comment:
This assumes that jabba is installed on the hosts. Which it is not on the older images.
an alternative would be to check presence of this shell script and install if needed:

```
if [ ! -f "${HOME}/.jabba/jabba.sh" ]; then
  curl ... | bash
fi
source "${HOME}/.jabba/jabba.sh"
```